doc/taskwiki.txt

```diff
@@ -604,6 +606,16 @@ selected tasks.
 *:TaskWikiMod* [mods]
     Opens a prompt for task modification, for selected task(s).
 
+*:TaskWikiOpen*
+    Opens annotations in the selected task(s) containing links to local files,
+    urls, etc.
+
+*:TaskWikiNote*
+    Adds a note to the task under cursor and opens it to be edit. This command
+    is compatible with the taskopen utility. See
```

Comment:
Would be nice to link to https://github.com/ValiValpas/taskopen

Reply:
Done.